tools/sst_scan.cc

```
@@ -0,0 +1,309 @@
+#include <stdlib.h>
+#include <libgen.h>
+
+#include "db/filename.h"
+#include "leveldb/env.h"
+#include "leveldb/db.h"
+#include "leveldb/cache.h"
+#include "leveldb/filter_policy.h"
+#include "leveldb/slice.h"
+#include "db/table_cache.h"
+#include "db/version_edit.h"
+#include "table/format.h"
+#include "table/block.h"
+#include "table/filter_block.h"
+
+//#include "util/logging.h"
+//#include "db/log_reader.h"
+
+void command_help();
+
+int
+main(
+    int argc,
+    char ** argv)
+{
+    bool error_seen, index_keys, all_keys, block_info, csv_header, counter_info,
+        running, no_csv;
+    int counter, error_counter;
+    char ** cursor;
+
+    running=true;
+    error_seen=false;
+
+    block_info=false;
+    counter_info=false;
+    index_keys=false;
+    csv_header=false;
+    all_keys=false;
+    no_csv=false;
+    counter=0;
+    error_counter=0;
+
+
```

Unusual, but confirmed with Matthew that it's NaB. ("./sst_stats -opt1 file -opt2" results in opt2 getting silently ignored. However, the intent is to support "./sst_stats -opt1 file -opt2 file2".)